.gitignore

```diff
@@ -1,4 +1,61 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
 node_modules/
-yarn.lock
-shrinkwrap.json
-package-lock.json
```

Comment:
I think the `yarn.lock` is required because travis-ci will use yarn when `yarn.lock` exists. ref: https://docs.travis-ci.com/user/languages/javascript-with-nodejs#yarn-is-supported

Reply:
Why use yarn on travis-ci? I don't like yarn at all..

Reply:
Okay, I found after I delete the `yarn.lock`, travis-ci still use `yarn test` command.